src/components/widgets/filesystem/FileSystemContextMenu.vue

```diff
@@ -16,11 +16,9 @@
         no-gutters
       >
         <v-col>
-          <v-list
-            dense
-          >
+          <v-list dense>
             <v-list-item
-              v-if="file.type !== 'directory' && rootProperties.accepts.includes('.' + file.extension) && rootProperties.canPrint"
+              v-if="canPrint"
```

Comment:
Not sure what the best behavior is here, but we can also just disable this if Klippy isn't ready instead of hiding it

Reply:
Indeed, only reason I opted for hiding is because we were doing the same for pre-heat! Having said that, it makes sense just to disable, so will change!

Reply:
Not sure "disabled" state is that visible... what you think?
![image](https://user-images.githubusercontent.com/85504/195932350-2103b13b-ab27-4420-953c-be5564ed8c4c.png)

Reply:
It looks fine IMO, but up to you ;)

Reply:
Updated to match the above!